packages/tags/stories/examples/Advanced.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { Story } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { Tag } from '@zendeskgarden/react-tags';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import { FauxInput } from '@zendeskgarden/react-forms';
+
+const tags = [
+  'veggies es bonus',
+  'beet greens corn soko endive',
+  'shallot courgette tatsoi pea sprouts',
+  'cauliflower',
+  'turnip yarrow ricebean rutabaga',
+  'celery quandong swiss chard chicory earthnut pea potato',
+  'grape wattle seed'
+];
+
+interface IAdvancedStoryProps {
+  size: 'small' | 'medium' | 'large';
+  isPill: boolean;
+  includeAvatar: boolean;
+  includeClose: boolean;
+  width: number;
+}
+
+export const Advanced: Story<IAdvancedStoryProps> = ({
```

Comment:
This story no longer demonstrates the functionality from the Styleguidist example. https://zendeskgarden.github.io/react-components/tags/#advanced
This is an important visual check to retain here.

Reply:
I've included the `width`, `size`, `pill`, `Tag.Avatar`, and `Tag.Close` options. What other controls need to be provided?

![image](https://user-images.githubusercontent.com/4030377/105106455-bb47a500-5a6a-11eb-84c5-752151d23750.png)

Reply:
Oh geez, I can't storybook 🤦 I was on the "Docs" tab.